tools/ci_build/build.py

```diff
@@ -934,10 +934,18 @@ def main():
             if args.test:
                 log.info("Cannot test on host build machine for cross-compiled ARM(64) builds. Will skip test running after build.")
                 args.test = False
-          else:
-            toolset = 'host=x64'
-            if (args.msvc_toolset):
-                toolset += ',version=' + args.msvc_toolset
+          else:            
+            if args.msvc_toolset == '14.16' and args.cmake_generator == 'Visual Studio 16 2019':
+                #CUDA 10.0 requires _MSC_VER >= 1700 and _MSC_VER < 1920, aka Visual Studio version in [2012, 2019)
```

Comment:
>CUDA 10.0 [](start = 17, length = 9)

Could we bump up CUDA to 10.1, so that we could start to use [cublasLt](https://docs.nvidia.com/cuda/cublas/index.html#using-the-cublasLt-api)?

Reply:
Sorry, no. I can't make the decision.

Reply:
I suggest to upgrade CUDA to 10.2 if possible. Of course, it could be in another pull request.

Reply:
@tianleiwu We can discuss it in our Scrum meeting.  We also want to upgrade the cudnn version as well, but we have to talk to the PMs on that.